pyresample/future/geometry/area.py

```diff
@@ -118,8 +118,8 @@ def __init__(
             "",
             area_id,
             crs,
-            width,
-            height,
+            shape[1],
+            shape[0],
```

Comment:
Should we throw a `NotImplementedError` if the size of shape is greater than 2?

Reply:
Oh, I like that. Good idea.

Reply:
Done.